demos/pedestrian_tracker_demo/cpp/CMakeLists.txt

```diff
@@ -5,11 +5,11 @@
 file(GLOB_RECURSE SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp ../${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 file(GLOB_RECURSE HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp ../${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 
-
-
+file(GLOB SRC_FILES ./*.cpp)
```

Comment:
Remove the two lines. They create a new variable `SRC_FILES` & `H_FILES` which are not used within this file.